components/PostCardList/PostCardList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  postType?: PostType;
+  className?: string;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
+}
+
+export default function PostCardList({
+  postType = PostType.recent,
+  children
+}: Props) {
+  return (
+    <Root>
+      <PostTypeContext.Provider value={postType}>
+        {children}
+      </PostTypeContext.Provider>
+    </Root>
+  );
+}
+
+const Root = styled.div`
+  display: grid;
+  grid-template-columns: 440px 440px;
```

Comment:
Please make components responsible. What it will show if the screen width is 960px, 1280px, or 1440px?